apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/HealthResource.java

```diff
@@ -22,13 +22,20 @@
 import jakarta.ws.rs.core.MediaType;
 import jakarta.ws.rs.core.Response;
 
+/** Legacy Health endpoint used by some routers to check if the service is up and running. */
 @Path("/")
-@Produces(MediaType.APPLICATION_JSON)
+@Produces(MediaType.TEXT_PLAIN)
 @Singleton
 public class HealthResource {
   @GET
-  @Path("/health")
+  @Path("health")
   public Response health() {
-    return Response.status(Response.Status.OK).entity("UP").build();
+    return Response.status(Response.Status.OK).entity("UP [REST]").build();
```

Comment:
I think various checkers do not validate contents, so for diagnostics/debugging purposes seems useful indicate source API. I am open to using some other notation but to make starting with "UP " makes sense for backwards compatibility.